### ISSUE TITLE
Peacock commands appear in palette but do nothing

### ISSUE THREAD

**Issue by tjeanes** [johnpapa/vscode-peacock#550]
**Versions (please complete the following information):**

- OS: Windows 11
- VS Code version: 1.89.1
- Peacock Version: 4.2.2

**Describe the bug**
Recently, the commands in the command palette have stopped working. My colours are still applied, but I cannot change them, list or edit my favourites, etc.

**To Reproduce**

1. Press F1 to open the Command Palette
2. Find any Peacock Command
3. Select it
4. The Command Palette closes, but nothing else happens

**Expected behavior**
The Peacock Command responds

**Screenshots**
![image](https://github.com/johnpapa/vscode-peacock/assets/113537509/862f4a3e-5dfd-46dc-9843-020cb0da3d96)
As you can see, my colours are still being applied and all the Peacock commands are listed. I've tried clicking each of them (including "Open the Documentation") and they all just close the command palette.

**Additional context**
I have tried the usual things: uninstalling and reinstalling the Extension, rebooting my PC, etc. These do not help.

I uninstalled Peacock and installed the "Ho-Oh" fork, and that worked as expected. My Peacock colours carried over and I could change them. I then uninstalled Ho-Oh and reinstalled Peacock. My changes persisted, but I still cannot use any Peacock command.

**Comment by ShrimpCryptid** [comment]
Bumping this as I'm also running into it! I'm on VS Code version 1.90.1, Windows 10.

**Comment by deshiknaves** [comment]
Try uninstalling and installing again. This fixed it for me.

**Comment by ShrimpCryptid** [comment]
> Try uninstalling and installing again. This fixed it for me.

Nope, still seeing the same behavior. I'll get the "Activating extensions" text at the bottom of the screen, but nothing happens. :/

![image](https://github.com/johnpapa/vscode-peacock/assets/30200665/f84865a4-7e97-459d-967b-26d5a8d5ecf2)

**Comment by Ralfaro17** [comment]
I had the same problem, but, as mentioned in [516](https://github.com/johnpapa/vscode-peacock/issues/516), uninstalling or disabling the live share extension fixed the problem for me

**Comment by tjeanes** [comment]
Interesting! Yes, disabling Live Share worked for me too.
I think it's worth noting that the Ho-Oh fork of Peacock does still work with Live Share enabled, so the clash with Live Share must be in some Peacock change since that fork.
Although disabling Live Share is a workaround, I won't mark this bug as closed just yet. I would like to be able to use both Peacock and Live Share together, and Ho-Oh demonstrates that this is still possible.

**Comment by diepes** [comment]
Ran into same problem with Peacock not working all of a sudden.
@tjeanes recommendation worked for me, i just disabled "Live Share"
Mac - vscode 1.96

**Comment by johnpapa** [comment]
Thank you for reporting this! Commands appearing but doing nothing could indicate an activation or workspace issue. Could you provide:
- VS Code version, Peacock version, OS
- Are you in a workspace (folder opened)? Peacock requires this.
- Output from the Peacock output channel after running a command
- Contents of `.vscode/settings.json`

Please also check if "Developer: Reload Window" resolves the issue.

**Comment by johnpapa** [comment]
Thanks for the detailed report and follow-up comments. I’m keeping this open pending updated diagnostics.

To move this to a fixable root cause, please provide this checklist from a current build:

- **VS Code version** (exact)
- **Peacock version** (exact)
- **OS**
- **Settings snippet**: relevant `peacock.*` and `workbench.colorCustomizations`
- **Remote type**: Local / WSL / SSH / Dev Container / Codespace
- **Repro steps**: minimal numbered steps in a fresh window
- **Diagnostics output**:
  - Peacock Output channel after `Developer: Reload Window`
  - `Log (Extension Host)` lines when invoking a Peacock command
  - Whether Live Share is installed/enabled at repro time

With this, we can determine if this is the same activation conflict pattern or a separate command execution bug.

---

Thanks again for reporting this. This is still a valid tracking issue for command execution failures.

AI-Ready next artifact needed: reproducible diagnostics on current builds (VS Code + Peacock versions, local/remote/container mode, extension host logs, and whether commands appear but no config updates occur).

Recommendation: keep open as canonical bug thread for command-palette execution failures.

**Comment by johnpapa** [comment]
Thank you to everyone who reported and investigated this — especially @tjeanes, @ShrimpCryptid, @ralfaro17, and @diepes for collectively pinpointing the root cause!

**Root cause (community-identified):** A conflict with the **VS Live Share** extension in certain states prevents Peacock command handlers from executing, even though the commands are visible in the palette.

**Workarounds (both confirmed by community reporters):**

1. **Disable or reinstall Live Share** — Disable the Live Share extension, run a Peacock command to confirm it works, then try reinstalling Live Share (uninstall → Developer: Reload Window → re-install).
2. **Verify a workspace folder is open** — Peacock needs a folder/workspace open to write to `.vscode/settings.json`. Run **Developer: Reload Window** after opening a workspace if commands still silently fail.

This is now documented in the FAQ: [Peacock commands appear but do nothing](https://johnpapa.github.io/vscode-peacock/guide/#peacock-commands-appear-but-do-nothing)

PR with the docs update: #674

Closing this as resolved by existing workaround + updated docs. If you're still seeing the issue on a recent build after trying the steps above, please open a new issue with the Peacock output channel log and extension host log — that will give us a fresh thread with the diagnostics needed to dig deeper.
